packages/synapse-react-client/package.json

```diff
@@ -113,7 +113,8 @@
     "ua-parser-js": "^1.0.35",
     "universal-cookie": "^4.0.4",
     "use-deep-compare-effect": "^1.8.1",
-    "xss": "^1.0.14"
+    "xss": "^1.0.14",
+    "zod": "^3.22.4"
```

Comment:
Recommend skimming https://zod.dev/, especially recommend reading the [Basic Usage](https://zod.dev/?id=basic-usage) section